Services/CO.CDP.Forms.WebApi/UseCase/UpdateFormSectionAnswersUseCase.cs

```diff
@@ -38,11 +38,22 @@ public async Task<bool> Execute((Guid formId, Guid sectionId, Guid answerSetId,
         }
```

Comment:
We also need to duplicate the AnswerSets/Answer, rather update when SharedConsent booking reference is already generated (not in draft mode)

Reply:
This is already discussed and not part of Dp-442

Reply:
Have we got a ticket for that? please add it to the backlog if not there, and add what was discussed.